### ISSUE TITLE
Pretrained models are missing

### ISSUE THREAD

**Issue by cantonioupao** [TMaysGGS/MTCNN-Keras#1]
Hello, 
Thank you for the implementation. Is it possible to provide also the pre-trained models of Pnet, Rnet and Onet as it will be easier and faster to just use them in detect_faces.py?  Thank you

**Comment by TMaysGGS** [comment]
Hi, 
Even a repo under MIT License can be used for any kind of purposes, the model trained with the dataset  FDDB/WIDERFace can not be published that way (Please refer to [https://github.com/ipazc/mtcnn/issues/53] for the detailed reason). And now I am suffering a potential bug when training PNet ([https://github.com/tensorflow/tensorflow/issues/38416]), which remains unsolved and turned me to use PyTorch. You can directly go to the original repo from the author [https://github.com/kpzhang93/MTCNN_face_detection_alignment] and transfer the weights from the original caffe model to keras model since they have the same structure except the classify output (I use sigmoid as a logic regression while the original paper uses softmax, and you might need to modify a little on the detect face scripts). And also the ipazc version [https://github.com/ipazc/mtcnn] also has very good performance.
